src/backend/rest.ts

```diff
@@ -72,10 +72,19 @@ export default class RestApiBackend implements Backend {
       url: '/version',
       method: 'GET',
     });
-    // TODO: check if this version matches the edition!
+    const checkMkVersion: string = result.data.versions.checkmk;
+    if (checkMkVersion.startsWith("2.0") || checkMkVersion.startsWith("1.6")) {
```

Comment:
```suggestion
    if (checkMkVersion.startsWith("2.0") || checkMkVersion.startsWith("1.")) {
```
why only exclude 1.6 and not other 1. versions?